source/js/main.js

```diff
@@ -83,154 +96,18 @@ let main = {
     envReq.send();
   },
 
-  fetchHomeDataIfNeeded(callback) {
-    // Only fetch data if you're on the homepage
-    if (document.querySelector(`#view-home`)) {
-      let homepageReq = new XMLHttpRequest();
-
-      homepageReq.addEventListener(`load`, () => {
-        callback.call(this, JSON.parse(homepageReq.response));
-      });
-
-      homepageReq.open(`GET`, `${networkSiteURL}/api/homepage/`);
-      homepageReq.send();
-    } else {
-      callback.call(this, {});
-    }
-  },
-
-  bindGlobalHandlers() {
-    // Track window scroll position and add/remove class to change sticky header appearance
-
-    let lastKnownScrollPosition = 0;
-    let ticking = false;
-    let elBurgerWrapper = document.querySelector(`.wrapper-burger`);
-
-    let adjustNavbar = scrollPosition => {
-      if (scrollPosition > 0) {
-        elBurgerWrapper.classList.add(`scrolled`);
-      } else {
-        elBurgerWrapper.classList.remove(`scrolled`);
-      }
-    };
-
-    let elCtaAnchor = document.querySelector(`#cta-anchor`);
-    let elStickyButton = document.querySelector(
-      `.narrow-sticky-button-container`
-    );
-    let noopCtaButton = () => {};
-    let adjustCtaButton = noopCtaButton;
-
-    if (elCtaAnchor && elStickyButton) {
-      let getAnchorPosition = () => {
-        return (
-          elCtaAnchor.getBoundingClientRect().top +
-          window.scrollY -
-          window.innerHeight
-        );
-      };
-
-      let ctaAnchorPosition = getAnchorPosition();
-
-      window.addEventListener(`resize`, () => {
-        ctaAnchorPosition = getAnchorPosition();
-      });
-
-      let scrollCtaButton = scrollPosition => {
-        if (scrollPosition > ctaAnchorPosition) {
-          elStickyButton.classList.add(`hidden`);
-          adjustCtaButton = noopCtaButton;
-        }
-      };
-
-      let initCtaButton = scrollPosition => {
-        if (scrollPosition <= ctaAnchorPosition) {
-          elStickyButton.classList.remove(`hidden`);
-          adjustCtaButton = scrollCtaButton;
-        }
-      };
-
-      adjustCtaButton = initCtaButton;
-    }
-
-    let onScroll = () => {
-      lastKnownScrollPosition = window.scrollY;
-
-      if (!ticking) {
-        window.requestAnimationFrame(() => {
-          adjustNavbar(lastKnownScrollPosition);
-          adjustCtaButton(lastKnownScrollPosition);
-          ticking = false;
-        });
-      }
-
-      ticking = true;
-    };
-
-    window.addEventListener(`scroll`, onScroll);
-
-    // Toggle sticky share buttons on blog page
-
-    let blogPageStickyButtons = document.querySelector(
```

Comment:
moved to source/js/foundation/template-js-handler/window/sticky-share-button-group.js